### PR TITLE
Change log and 6.0.0-alpha.12 version number updates

### PR DESCRIPTION
**Pull request by mirichmo**

### CODE REVIEW

**Comment by mirichmo** [comment]
@daxian-dbw @andschwa Please Review

**Comment by mirichmo** [comment]
Travis is failing because the v6.0.0-alpha.12 package has not been published on GitHub.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,26 @@ Changelog
...
+- Fix GetParentProcess() to replace an expensive WMI query with Win32 API calls
+- Fix `Set-Content` failure to create a file in PSDrive under certain conditions.
+- Adding ValidateNotNullOrEmpty to `-Name` parameter of `Get-Service`
+- Adding support <Suppress> in `Get-WinEvent -FilterHashtable`
+- Adding WindowsVersion to `Get-ComputerInfo`
+- Remove the unnecessary use of lock in PseudoParameterBinder to avoid deadlock
+- Refactor `Get-WinEvent` to use StringBuilder for XPath query construction
+- Clean up and fix error handling of libpsl-native
+- Exclude Registry and Certificate providers from UNIX PS
+- Update PowerShell Core to consume DotNet Core preview1-24530-04
```
Comment (andschwa) [inline CHANGELOG.md]: .NET Core

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,26 @@ Changelog
 Unreleased
 ----------
 
-v6.0.0-alpha.12 - 2016-10-31
+v6.0.0-alpha.12 - 2016-11-02
```
Comment (andschwa) [inline CHANGELOG.md]: 03

**Review by andyleejordan — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (mirichmo) [inline CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 1 above)
Comment (mirichmo) [inline CHANGELOG.md]: Fixed

**Comment by daxian-dbw** [comment]
Hi @andschwa, I just noticed this morning that the `docker` branch in `PowerShell/PowerShell` is gone. Isn't the automated docker build at https://hub.docker.com/r/microsoft/powershell/ depending on it?
Was that branch removed on purpose?

**Comment by andschwa** [comment]
@daxian-dbw I have no clue why it was removed. But it's not matter, just make another and push it.

**Comment by daxian-dbw** [comment]
@andschwa OK, will do. And that will trigger the automated docker build of `microsoft/powershell` right?

**Comment by andschwa** [comment]
Correct, but the packages need to be posted first. Love chicken-egg problems :)

**Comment by daxian-dbw** [comment]
@andschwa Got it. Thanks!

**Pull request merged**